paasta_tools/kubernetes_tools.py

```diff
@@ -1378,7 +1378,7 @@ def get_volume_mounts(
             if secret_hash:
                 mount = V1VolumeMount(
                     mount_path="/etc/boto_cfg",
-                    name=f"secret-boto-key-{service_name}",
+                    name=limit_size_with_hash(f"secret-boto-key-{service_name}"),
```

Comment:
I think for the volume names here (the V1Volume and the V1VolumeMount) we want to use [get_secret_volume_name](https://sourcegraph.yelpcorp.com/Yelp/paasta@f26daac26d5edd2091e2c75ff7f29b5c2e81a0f4/-/blob/paasta_tools/kubernetes_tools.py?subtree=true#L835:1) so that we ensure that we do all the other normalization stuff to comply with the restrictions on what volume names can look like